package.json

```diff
@@ -229,9 +229,8 @@
     "watch": "tsc -watch -p ./ && node scripts/copyVendor.js",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run compile && node ./node_modules/vscode/bin/test",
-    "check": "gts check",
-    "clean": "gts clean",
-    "fix": "gts fix",
+    "check": "./node_modules/eslint/bin/eslint.js . --ext .js,.jsx,.ts,.tsx",
+    "fix": "./node_modules/eslint/bin/eslint.js . --ext .js,.jsx,.ts,.tsx --fix",
```

Comment:
can we merge check and fix into one script since they are always called at the same time? name the script as "eslint"?

Reply:
The two script do different things. If you just want to check you can run `npm run check` which will execute eslint check. We use this command in CI build.
If you would like to auto fix the eslint error while checking them, run the `npm run fix`. Developers can leverage this command to auto-fix eslint error while developing.